### PR TITLE
Support launching Drasi Tutorial in Github Codespaces

### PR DESCRIPTION
**Pull request by amansinghoriginal**
# Description
- Add `.devcontainer/devcontainer.json` file to serve as default codespace configuration. This launches getting started tutorial.
- Update the README.md file for Getting Started tutorial to provide link to Drasi Docs and separate out the VS Code steps.

### CODE REVIEW

File: tutorial/getting-started/README.md
Comment (ruokun-niu) [inline tutorial/getting-started/README.md:1]: Do we need to add a link to the tutorial codespace in the root readme?

**Comment by danielgerlag** [comment]
If we wanted to add more tutorials later, that perhaps each use different dependencies (eg. Postgres, MySQL, etc...)
What would that look like?

**Comment by amansinghoriginal** [comment]
> If we wanted to add more tutorials later, that perhaps each use different dependencies (eg. Postgres, MySQL, etc...) What would that look like?

Check the [discussion here](https://teams.microsoft.com/l/message/19:092b2a661abe4f328a0d22ffc4638bae@thread.tacv2/1729787029088?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=2788cfe9-d00f-4ef5-b239-02c56ab06f2f&parentMessageId=1729716489976&teamName=Project%20Drasi%20Core%20Team&channelName=Engineering%20Team&createdTime=1729787029088&ngc=true).

To summarize, by default we will launch the codespace with config for getting started tutorial.
For other apps, we can create separate devcontainer config files, and user can select option `New with Options` from the Codespaces menu as shown in screenshot:
image

**Review by danielgerlag — APPROVED** [review]

File: tutorial/getting-started/README.md
Comment (amansinghoriginal) [reply tutorial/getting-started/README.md:1]: @ruokun-niu Added a link to getting started tutorial and a button to launch codespace.

File: README.md [hunk 1]
```diff
@@ -1,8 +1,13 @@
...
+This repo contains code for a tutorial and a few sample applications that use Drasi as an end-to-end solution for change detection and real-time reactions. The tutorial The applications will help you understand how to implement and deploy Drasi for different scenarios. Each app folder within this repo has a README.md which details what the scenario for the app, code for the apps, requisite test data, and how to run the samples.
+
+## Getting Started with Drasi
+#### Detect and react to your first database change using Drasi
+[This step-by-step tutorial](https://drasi.io/getting-started/) will help you get Drasi up and running quickly and show you how easy it is to create Sources, Continuous Queries, and Reactions.
+[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/drasi-project/learning)
```
Comment (amansinghoriginal) [inline README.md:7]: @agentofreality Here is a link to launch codespace that we can include in various places including drasi.io.

**Review by ruokun-niu — APPROVED** [review]

**Pull request merged**